app/controllers/users_controller.rb

```diff
@@ -8,22 +8,32 @@ def show
 
   # GET /users/1/edit
   def edit
+    @tab_select = params[:tab]
+    if @tab_select.present?
```

Comment:
Please don't do that with AJAX. Just use the bootstrap tabs to hide and show the different partials.